src/components/Line/index.tsx

```diff
@@ -67,7 +67,13 @@ export default class Line extends Component<any, any> {
         return (
             <div className={classes} style={lineStyle}>
                 <LineNumber number={number} highlight={highlight} onClick={onLineNumberClick} />
-                <LineContent number={number} onClick={onRowClick} formatPart={formatPart} data={data} />
+                <LineContent
+                    number={number}
+                    formatPart={formatPart}
+                    data={data}
+                    onMouseUp={onMouseUp}
+                    style={{ width: "100%", display: "inline-block" }}
```

Comment:
Looks like this would get passed through as a prop now? Might want to test if this was needed or not?